packages/office-ui-fabric-react/src/utilities/grid/Grid.types.ts

```diff
@@ -50,4 +57,34 @@ export interface IGridProps {
    * The optional size of the parent set (size of parent menu, for example)
    */
   setSize?: number;
+
+  /**
+ * Optional styles for the component.
+ */
+  getStyles?: IStyleFunction<IGridStyleProps, IGridStyles>;
+}
+
+/**
+ * Properties required to build the styles for the grid component.
+ */
+export interface IGridStyleProps { }

```

Comment:
add theme

Reply:
Sure!! let me do that :) Thanks for looking at this.